### PR TITLE
Updated responding to feedback

### PR DESCRIPTION
**Pull request by rayhm6419**

### CODE REVIEW

File: src/components/home/Chart/IncidentChart.js [hunk 1]
```diff
@@ -108,39 +111,54 @@ const IncidentChart = ({ color, chart_data, state, isFirstLoadData }) => {
...
         </>
       ) : null}
-      <ResponsiveContainer >
+
+      <ResponsiveContainer>
       <ComposedChart 
-            height={300}
-            width={794} 
+            height={382}
+            width={1000}
```
Comment (lima01) [inline src/components/home/Chart/IncidentChart.js:118]: Hardcode the width make the chart not responsive to window size change. Can we follow the main branch, only set the height?

**Comment by lima01** [comment]
Please include public/locales/en folder and other files to show the chart content and tooltip content properly.

**Review by lima01 — CHANGES_REQUESTED** [review]

**Comment by lima01** [comment]
Hi 
[@Haoming Lei](https://techforaapi.slack.com/team/U051GU54G2G)
 thank you for the great improvement you made to the home page. Here are a few things  I noticed in the latest version:

- Font in the yellow box is sans-serif, the design is Open Sans.
- The chart will break if you resize the screen. I would recommend to follow the main branch to make it responsive.
- The chat tooltip is showing translation key, need to include the i18n related files to show proper english text.
- There are more blank spaces in various components, such as the yellow boxes, the center area, etc.
- Font for the buttons look different from the design
- Might need more work on responsive design. When screen is small, many compoents don't align well anymore.

Thanks!

File: src/components/home/Chart/IncidentChart.js (hunk 1 above)
Comment (rayhm6419) [reply src/components/home/Chart/IncidentChart.js:118]: Thank you for your attention the chart is not responsive to window size changes, I will fix it followed by the main branch.

**Review by lima01 — APPROVED** [review]

**Pull request merged**